uamqp/error.py

```diff
@@ -248,3 +248,25 @@ def __init__(self, status_code, status_description):
         if self.description:
             message += u"\nDescription: {}".format(self.status_description.decode('utf-8'))
         super(TokenAuthFailure, self).__init__(message)
+
+
+class ErrorResponse(object):
+    """
+    """
+    def __init__(self, error_info=None, condition=None, description=None, info=None):
+        info = None
+        self.condition = condition
+        self.description = description
+        self.info = info
+        self.error = error_info
+        if isinstance(error_info, list) and len(error_info) >= 1:
+            if isinstance(error_info[0], list) and len(error_info[0]) >= 1:
+                self.condition = error_info[0][0]
+                if len(error_info[0]) >= 2:
+                    self.description = error_info[0][1]
+                if len(error_info[0]) >= 3:
+                    info = error_info[0][2]
+        try:
+            self.info = info.value
```

Comment:
Will this actually ever have a `.value` attribute? Or is that a symptom of the old c-amqp types?

Reply:
good catch, I copied the code from uamqp, the `.value` should not be here.